packages/ui/src/hooks/use-apple-auth.ts

```diff
@@ -0,0 +1,92 @@
+import camelcaseKeys from 'camelcase-keys';
+import { useNavigate } from 'react-router-dom';
+
+import { inOperator } from '@/utils';
+
+export const loadAppleSdk = async () =>
+  new Promise((resolve, reject) => {
+    const script = document.createElement('script');
+
+    script.addEventListener('load', resolve);
```

Comment:
Do we need to `removeEventListener` after the script is loaded or failed?

Reply:
i think it's ok, it'll trigger just once